doc/source/methods/PartialDependenceVariance.ipynb

```diff
@@ -0,0 +1,432 @@
+{
```

Comment:
<ul><li>vAs the naming suggest -&gt; As the naming suggests</li><li>hot(i.e. -&gt; hot (i.e.  // (add space)</li><li>Worth mentioning that the model-internal method based on impurity is because it's a tree ensemble (random forest?) model?</li><li>standardize procedure&nbsp;-&gt; standardized procedure</li><li>some internal feature importance -&gt; some internal feature importance notions</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>